### PR TITLE
Fix some issues and simplify the FCM

### PR DESCRIPTION
**Pull request by jimmynotjim**
The layout code for the Featured Content Module was overly complex resulting in magic numbers that needed to be updated by hand with other layout changes. Updated to remove the magic numbers as much as possible and fix some visual bugs currently being addressed in cfgov-refresh.

## Changes

- Simplified the FCM layout styles

## Testing

In this repo:
- check out this branch
- make sure you've run `npm run cf-link`

In a site specific clone of this repo, 
- make sure you're on the `gh-pages` branch
- run `npm run cf-link`
- run `npm run build`
- run `npm start`
- navigate to `http://localhost:3000/components/cf-layout/#featured-content-module`

## Review

- @sebworks 
- @mistergone 
- @Scotchester 
- @cfarm 
- @anselmbradford 

## Screenshots

Default (Left Aligned):

screen shot 2017-05-25 at 3 05 09 pm

Right Aligned:

screen shot 2017-05-25 at 3 05 04 pm

Centered:

screen shot 2017-05-25 at 3 04 58 pm

## Notes

- Note that the `block` classes have be separated completely from the FCM. We've found that mixing global layout classes like `block` with atomic components often leads to trouble and it's better to separate each of the concerns.

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/cf-layout/src/organisms/featured-content-module.less [hunk 1]
```diff
@@ -1,88 +1,69 @@
...
       overflow: hidden;
-    } );
 
-    .respond-to-min( @bp-med-min, {
-      left: 330px;
+      .lt-ie10 & {
+        position: absolute;
+        top: 0;
+        right: 0;
+        left: 360px; // widht of 5 of 12 cols at lg size + grid gutters
```
Comment (anselmbradford) [inline src/cf-layout/src/organisms/featured-content-module.less]: `width`

Comment (jimmynotjim) [inline src/cf-layout/src/organisms/featured-content-module.less]: Oh no, I think I made the same typo in cfgov-refresh, thanks for catching it.

**Comment by jimmynotjim** [comment]
@anselmbradford Looks like we're not adding the `lt-ie10` class on the docs site. Try testing `https://www.consumerfinance.gov/about-us/the-bureau/`, cfgov-refresh is using the same code.

**Comment by anselmbradford** [comment]
> Looks like we're not adding the lt-ie10 class setting autoprefixer correctly on the docs site so the -ms- prefix is missing from the transform. Try testing https://www.consumerfinance.gov/about-us/the-bureau/, cfgov-refresh is using the same code.

Okay, looks fine there.

**Pull request merged**